### ISSUE TITLE
Build from source launch the server 

### ISSUE THREAD

**Issue by hjh0119** [carla-simulator/carla#6551]
Is it possible to launch the server exclusively using "make launch"?Additionally, how to launch the server, built from source, using Python, both locally and remotely?

CARLA version: 0.9.14
Platform/OS: Ubuntu 20.04

**Comment by Blyron** [comment]
Yes, make launch and press play. Then you can use python exmaple scripts
